docs/src/basics/built_in_types.md

```diff
@@ -1,8 +1,8 @@
 # Built-in Types
 
-Every value in Sway is of a certain type. Although deep down, all values are just ones and zeroes in silicon, Sway needs to know what those ones and zeroes actually mean. This is accomplished with _types_.
+Every value in Sway is of a certain type. Although deep down, all values are just ones and zeroes in the underlying virtual machine, Sway needs to know what those ones and zeroes actually mean. This is accomplished with _types_.
```

Comment:
_the underlying virtual machine which is then on a real machine predicated upon silicon_


just kidding